packages/contracts-bedrock/foundry.toml

```diff
@@ -20,7 +20,7 @@ remappings = [
   'ds-test/=lib/forge-std/lib/ds-test/src',
   'safe-contracts/=lib/safe-contracts/contracts',
   'kontrol-cheatcodes/=lib/kontrol-cheatcodes/src',
-  'solady/=lib/solady/src',
+  '@solady/=lib/solady/src',
```

Comment:
Tiny nit (sorry for OCD) - can we move this under the other deps w/ `@` prefixes?

Reply:
updated